src/openssl/lib_crypto.cr

```diff
@@ -106,10 +106,13 @@ lib LibCrypto
 
   fun BIO_new(BioMethod*) : Bio*
   fun BIO_free(Bio*) : Int
-  fun BIO_set_data(Bio*, Void*)
-  fun BIO_get_data(Bio*) : Void*
-  fun BIO_set_init(Bio*, Int)
-  fun BIO_set_shutdown(Bio*, Int)
+
+  {% if compare_versions(LibCrypto::OPENSSL_VERSION, "1.1.0") >= 0 %}
+    fun BIO_set_data(Bio*, Void*)
+    fun BIO_get_data(Bio*) : Void*
+    fun BIO_set_init(Bio*, Int)
+    fun BIO_set_shutdown(Bio*, Int)
+  {% end %}
 
   {% if compare_versions(LibCrypto::OPENSSL_VERSION, "1.1.0") >= 0 %}
```

Comment:
```suggestion

```

Reply:
I intentionally left the checks there to separate them into groups, but it does increase the clutter level.

Reply:
Wouldn't empty line do the trick?

Reply:
Yes, yes it would :-) Should I add it?

Reply:
Yes, please :)